home_work_23/ViewController.swift

```diff
@@ -8,11 +8,18 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    var valueKey: String?
     override func viewDidLoad() {
         super.viewDidLoad()
         // адрес сервера
-        if let url = URL(string: "https://api.openweathermap.org/data/2.5/weather?q=London&appid=3b0d514cd0fcc25c0056ef824e4464ad") {
+        let appDelegate = UIApplication.shared.delegate as? AppDelegate
```

Comment:
Why `appDelegate`?
It's not a requirement create and variables into `appDelegate` and use it later.
You could create a computed property which will read from `info.plist`, for example.